app/scripts/components/alerts.component.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .component('alerts', {
+    bindings: {
+      // Map (or array) of alert objects, each with properties
+      //   type:     'error', 'warning', 'success', or 'info' (default)
+      //   message:  the message to display
+      //   links:    an array of link objects (optional, see below)
+      //   onClose:  callback function when the user dismisses the alert (optional)
+      //
+      // Each link can have the following properties:
+      //   href:     link URL (optional if onClick is set)
+      //   label:    the link text
+      //   onClick:  handler to call when the link if clicked (optional)
+      //             If onClick returns true, dismisses the alert
+      alerts: '=',
+      // Filter function (optional).
+      filter: '=?',
+      animateSlide: '=?',
+      hideCloseButton: '=?',
+      toast: '=?'
```

Comment:
I think we can remove this now. @TheRealJon can you sweep the code to check if it's still used anywhere?

Can you also check `animateSlide`?